test/test_core.py

```diff
@@ -5942,9 +5942,8 @@ def test_istream(self):
       self.set_setting('LINKABLE', linkable)
       self.do_core_test('test_istream.cpp')
 
-  def test_fs_dir_wasmfs(self):
-    self.emcc_args += ['-sWASMFS']
-    self.emcc_args += ['-sFORCE_FILESYSTEM']
+  def test_fs_dir(self):
+    self.set_setting("FORCE_FILESYSTEM")
```

Comment:
This is a good change, but after it WasmFS is not tested on CI, because we test `test_fs_dir` automatically as part of the core tests, but we don't run all of `wasmfs.*`, so we don't run `wasmfs.test_fs_dir`. Please add this to `.circleci/config.yml` to fix that.